src/backend/base/langflow/schema/dataframe.py

```diff
@@ -103,3 +137,31 @@ def __bool__(self):
         Returns True if the DataFrame has at least one row, False otherwise.
         """
         return not self.empty
+
+    def to_lc_documents(self) -> list[Document]:
+        """Converts the DataFrame to a list of Documents.
+
```

Comment:
We should turn into Data (i.e `to_data_list`) then to docs so that we have a unified way of doing this.